Client/src/Views/UserDatasets/UserDatasetsWorkspace.tsx

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import WorkspaceNavigation from "wdk-client/Components/Workspace/WorkspaceNavigation";
+import { Route, Switch, Redirect } from "react-router";
+import UserDatasetListController from "wdk-client/Controllers/UserDatasetListController";
+import UserDatasetNewUploadController from "wdk-client/Controllers/UserDatasetNewUploadController";
+import { UserDatasetAllUploadsController } from "wdk-client/Controllers";
+import { useWdkService } from "wdk-client/Hooks/WdkServiceHook";
+import HelpIcon from "wdk-client/Components/Icon/HelpIcon";
+
+interface Props {
+  rootPath: string;
+};
+
+function UserDatasetsWorkspace(props: Props) {
+  const config = useWdkService(wdkService => wdkService.getConfig());
+  const { rootPath } = props;
+  if (config == null) return null;
+  if (config.projectId !== 'MicrobiomeDB') return (
```

Comment:
If `projectId` works as a key, it works, but I remember comparing on `displayName`